doc] Fix inconsistencies with torch.linalg.matrix_rank doc

heitorschueroff opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #51672 [doc] Fix inconsistencies with torch.linalg.inv and deprecate torch.inverse
* #51671 [doc] Fix inconsistencies with linalg.pinv docs and deprecate pinverse
* **#51660 doc] Fix inconsistencies with torch.linalg.matrix_rank doc**
* #51659 [doc] Fix inconsistencies with torch.linalg.eigvalsh
* #51658 [doc] Fix inconsistencies with torch.linalg.eigh
* #51651 [doc] Fix inconsistencies with torch.linalg.det docs
* #51641 [doc] Fix inconsistencies with torch.linalg.cond doc

Differential Revision: [D26234100](https://our.internmc.facebook.com/intern/diff/D26234100)

heitorschueroff committed:
doc] Fix inconsistencies with torch.linalg.matrix_rank doc

[ghstack-poisoned]

heitorschueroff committed:
Update on "doc] Fix inconsistencies with torch.linalg.matrix_rank doc"




[ghstack-poisoned]

mruberry reviewed (approved):
Just one batch language tweak.

heitorschueroff committed:
Update on "doc] Fix inconsistencies with torch.linalg.matrix_rank doc"


Differential Revision: [D26234100](https://our.internmc.facebook.com/intern/diff/D26234100)

[ghstack-poisoned]

heitorschueroff committed:
Update on "doc] Fix inconsistencies with torch.linalg.matrix_rank doc"


Differential Revision: [D26234100](https://our.internmc.facebook.com/intern/diff/D26234100)

[ghstack-poisoned]

heitorschueroff committed:
Update on "doc] Fix inconsistencies with torch.linalg.matrix_rank doc"


Differential Revision: [D26234100](https://our.internmc.facebook.com/intern/diff/D26234100)

[ghstack-poisoned]

heitorschueroff committed:
Update on "doc] Fix inconsistencies with torch.linalg.matrix_rank doc"


Differential Revision: [D26234100](https://our.internmc.facebook.com/intern/diff/D26234100)

[ghstack-poisoned]

heitorschueroff committed:
Update on "doc] Fix inconsistencies with torch.linalg.matrix_rank doc"


Differential Revision: [D26234100](https://our.internmc.facebook.com/intern/diff/D26234100)

[ghstack-poisoned]
